storage/s3-sdk/list_gcs_buckets.py

```diff
@@ -21,11 +21,19 @@
 
 
 def list_gcs_buckets(google_access_key_id, google_access_key_secret):
-    """Lists all GCS buckets using boto3 SDK"""
-    # Create a new client and do the following:
-    # 1. Change the endpoint URL to use the
-    #    Google Cloud Storage XML API endpoint.
-    # 2. Use Cloud Storage HMAC Credentials.
+    """Lists all Cloud Storage buckets using AWS SDK for Python (boto3)
+    Positional arguments:
+    google_access_key_id          -- hash-based message authentication code (HMAC) access ID
+    google_access_key_secret      -- HMAC access secret
+
+    Returned value is a list of strings, one for each bucket name.
+
+    To use this sample:
+    1. Create a Cloud Storage HMAC key: https://cloud.google.com/storage/docs/authentication/managing-hmackeys#create
+    2. Change endpoint_url to a Google Cloud Storage XML API endpoint.
+
+    To learn more about HMAC: https://cloud.google.com/storage/docs/authentication/hmackeys#overview
+    """
```

Comment:
Please follow the docstring format as shown in https://google.github.io/styleguide/pyguide#383-functions-and-methods.
```suggestion
    """Lists all Cloud Storage buckets using AWS SDK for Python (boto3)

    Positional arguments:
    google_access_key_id          -- hash-based message authentication code (HMAC) access ID
    google_access_key_secret      -- HMAC access secret

    To use this sample:
    1. Create a Cloud Storage HMAC key: https://cloud.google.com/storage/docs/authentication/managing-hmackeys#create
    2. Change endpoint_url to a Google Cloud Storage XML API endpoint.

    To learn more about HMAC: https://cloud.google.com/storage/docs/authentication/hmackeys#overview
    
    Returns:
        A list of strings, one for each bucket name.
    """
```